app.js

```diff
@@ -137,10 +144,62 @@ function handleImagesOnClick(event) {
 
   if (totalClicks >= 25) {
     imgContainer.removeEventListener('click', handleImagesOnClick);
+    //connected to function displayFinalResults up top
     displayFinalResults();
+    createChart();
   }
 }
 
+var imgNamesChart = [];
+var clicksForChart = [];
+var viewsForChart = [];
+
+function createChartData (){
+  for (var i = 0; i < allProducts.length; i++) {
+    imgNamesChart.push(allProducts[i].name);
+    clicksForChart.push(allProducts[i].clicks);
+    viewsForChart.push(allProducts[i].views);
+  }
+  console.log(imgNamesChart);
```

Comment:
Nice log statements! It may be clearer which log statement is which if you do something like 

```javascript
console.log('imgNamesChart: ', imgNamesChart);
```